slidechain/submit.go

```diff
@@ -124,6 +126,16 @@ func (s *submitter) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 	wait := (waitStr != "")
 
+	blockIntervalStr := req.FormValue("blockInterval")
```

Comment:
Do we want anyone who submits a transaction to be able to set the block interval? Seems potentially dangerous